src/components/layer-controller/reducer.js

```diff
@@ -35,16 +35,33 @@ function channelsToLayerProps(channels) {
   };
 }
 
+function getNewChannelProperty(channel, property, value) {
+  if (property === 'visibility') {
+    return !channel.visibility;
+  }
+  if (property === 'selection') {
+    return { ...channel[property], ...value };
+  }
+  return value;
+}
+
```

Comment:
Allows us to have more than field in the selection.

Reply:
ok cool